prow/plugins/override/override.go

```diff
@@ -441,11 +443,13 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent,
 
 	if unknown := overrides.Difference(contexts); unknown.Len() > 0 {
 		resp := fmt.Sprintf(`/override requires a failed status context, check run or a job name to operate on.
-The following unknown contexts were given:
+The following unknown contexts/checkruns were given:
+%s
+
+Only the following contexts/checkruns were expected:
 %s
 
-Only the following contexts were expected:
-%s`, formatList(unknown.List()), formatList(contexts.List()))
+Successful checkruns are listed but can not be overridden.`, formatList(unknown.List()), formatList(contexts.List()))
```

Comment:
Shouldn't we exclude them?

Reply:
No, because succesful statuses are listed too

Reply:
Right and that is wrong, isn't it? Why would we do that for checkruns and not for contexts? It makes way more sense to filter those out above or not?

Reply:
Ah, I can modify it to remove successful checkruns and contexts from the list that goes on the comment.

Reply:
Fixed